src/Stripe.net/Entities/OAuth/OAuthDeauthorize.cs

```diff
@@ -4,13 +4,12 @@ namespace Stripe
 
     public class OAuthDeauthorize : StripeEntity<OAuthDeauthorize>
     {
+        /// <summary>
+        /// The unique id of the account you have revoked access to, as a string. This is the same
+        /// as the <see cref="OAuthDeauthorizeOptions.StripeUserId"/> you passed in. If this is
+        /// returned, the revocation was successful.
+        /// </summary>
         [JsonProperty("stripe_user_id")]
         public string StripeUserId { get; set; }
-
-        [JsonProperty("error")]
-        public string Error { get; set; }
-
-        [JsonProperty("error_description")]
-        public string ErrorDescription { get; set; }
```

Comment:
is this a breaking change?

Reply:
It isn't, as those attributes were never used. When an OAuth error occurs, the library raises a `StripeError` with the `Error` and `ErrorDescription` filled.